server/events/command/comment.go

```diff
@@ -48,6 +69,8 @@ type Comment struct {
 	ProjectName string
 	// LogLevel is the name log level verbosity requested on the underlying Terraform operation.
 	LogLevel string
+
+	PolicyCheckOutput PolicyCheckOutput
```

Comment:
This seems like a random thing to add to this object which specifically describes a comment command.